### ISSUE TITLE
Missing CSS and JS files?

### ISSUE THREAD

**Issue by bsvensson** [Esri/calcite-bootstrap#296]
It seems like https://esri.github.io/calcite-bootstrap/assets/css/calcite-bootstrap-open.min.css  and https://esri.github.io/calcite-bootstrap/assets/js/bootstrap.min.js  disappeared recently. 
* We use the first on in https://developers.arcgis.com/javascript/latest/sample-code/portalitem-dragndrop/index.html.
* We use both in https://developers.arcgis.com/javascript/3/samples/portal_embed_in_website/

Are there new URLs/locations for these?  (Or should we not use them?)

**Comment by ajturner** [comment]
Ping @FelFly

**Comment by FelFly** [comment]
@bsvensson Yes. We did remove them because this project is no longer running a full instance of Bootstrap and I needed the gh-pages branch to write updated documentation (which I should have sometime this week.) Full story here: https://github.com/Esri/calcite-bootstrap/issues/294. 

If you have a copy of the javascript files and want to make a PR to add them to the assets folders of the docs like we did for umacgillivray with the CSS files, we can probably add that as a holdover for the old version of this project.

But moving forward, I'm trying to make this into an actual theme that you use on top of the vanilla Bootstrap code - that way folks can have Calcite Bootstrap or customer themes applied and without running two sets of Bootstrap files.

**Comment by ajturner** [comment]
@klara it would also be helpful if you can write an example of how the library should be used. Show loading bootstrap and then calcite theme.

It might also be helpful to have a "merged version" so it's a single HTTP load

**Comment by FelFly** [comment]
@ajturner Yes. I have all this written in the calcite-bootstrap-theme repo. I just need to move it over here.

As for a merged version - I don't know what you mean. If you mean that we should have a set of Bootstrap files in this project, I was specifically told by three people in two teams that was the incorrect way to handle a theme and I should remedy it with this version. With this version, there are three paths to running this theme:
1) Run Calcite Bootstrap as strictly Bootstrap variables
2) Run Calcite Bootstrap as Bootstrap variables + expansion variables/CSS to achieve stricter Calcite Web "look" and to have support for card components
3) Run either one of the patterns above, but in the event of a shared theme or customer theme, use that instead

**Comment by mjuniper** [comment]
It seems to me that the consuming app's build process should produce a single css file that is bootstrap + calcite bootstrap + any other css the app needs.

**Comment by ajturner** [comment]
Let's presume that most people who create websites don't have a "build process" - instead they "edit HTML and JS files". And it may even be a static website <gasp>

My assumption is that describes most "Devs". Just try showing any new dev how to start a new project and do builds.

Regardless, we need some "step-by-step" docs in the README for someone who is building a simple, static website.
